pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -479,11 +493,13 @@ func (s *Server) createIstioCA(opts *caOptions) (*ca.IstioCA, error) {
 	return istioCA, nil
 }
 
-func (s *Server) createSelfSignedCACertificateOptions(fileBundle *ca.SigningCAFileBundle, opts *caOptions) (*ca.IstioCAOptions, error) {
+// loadCAFromSecrets will initialize the Istio CA using istio-ca-secrets or cacerts.
+// This is only called if mounted files
```

Comment:
not understand it

Reply:
Sorry, got interrupted in the middle and forgot to finish the comment :-)

Will add the rest "if mounted files are not found".

Reply:
This could be called if `features.UseCacertsForSelfSignedCA && istioGenerated` is true. In this case the `cacerts` secret is mounted.